service-wcs/src/main/java/org/oskari/wcs/parser/CapabilitiesParser.java

```diff
@@ -0,0 +1,197 @@
+package org.oskari.wcs.parser;
+
+import static java.util.stream.Collectors.groupingBy;
+import static java.util.stream.Collectors.mapping;
+import static java.util.stream.Collectors.toList;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.xml.parsers.ParserConfigurationException;
+import org.oskari.wcs.capabilities.BoundingBox;
+import org.oskari.wcs.capabilities.Capabilities;
+import org.oskari.wcs.capabilities.Contents;
+import org.oskari.wcs.capabilities.CoverageSummary;
+import org.oskari.wcs.capabilities.Operation;
+import org.oskari.wcs.capabilities.OperationsMetadata;
+import org.oskari.wcs.capabilities.ServiceIdentification;
+import org.oskari.wcs.capabilities.ServiceMetadata;
+import org.oskari.wcs.util.WCS;
+import org.oskari.wcs.util.XML;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+public class CapabilitiesParser {
+
+    public static final String ALLOWED_VERSION = "2.0.1";
```

Comment:
Should this be used in all the code for WCS here having the "2.0.1" version as string to make it easier to find where version references are?